content/material_and_questions.md

```diff
@@ -13,23 +13,31 @@
 | 2026-2027 | John                                          | Narrative |
 | 2027-2028 | Hebrews, 1 Peter, 2 Peter                     | Epistle   |
 
-## Question Writing
+## Material
+: The biblical content that is the basis for competition for the quiz year. This includes Book Name, Chapter Number, and Verse Number. This does not include any specifically excluded material.
 
-### Preparation of Questions
+Of the eight years of material, 4 are defined as Narrative and 4 are defined as Epistle.
 
-### Question Types
+## Interrogative Words
```

Comment:
Structurally, this flow jumps from material into an interrogative word, which is a pretty large context shift. I'd start with material, then talk about the concept of questions at a general level, then dive into concepts like interrogative words, unique words/phrases, and so on.

Reply:
Yeah, I get it. There's just not much to define regarding "Questions." I just threw something in there though.